src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationServiceTest.java

```diff
@@ -35,7 +35,7 @@
     @Captor
     private ArgumentCaptor<EmailData> emailDataArgument;
 
-    private Claim claim;
+    private Claim claimWithFullDefenceResponse;
```

Comment:
Can we move it to test methods? Now since we added more tests we this field is not used for each tests and as such should not be prepared in `beforeEachTest` method.